packages/core/schema/lavamoat-policy.v0-0-1.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "LavaMoat Policy Schema",
+  "$id": "lavamoat-policy.v0-0-1",
+  "type": "object",
+  "description": "Schema for LavaMoat policy files",
+  "properties": {
+    "resources": {
+      "description": "Describe the resources available to your application and direct dependencies",
+      "title": "Resources",
+      "type": "object",
+      "$comment": "This one or more valid npm package names delimited by a \">\" character",
```

Comment:
words seem to be missing here

I'd say `Keys are canonical paths to dependencies - one or more valid npm package names joined by a \">\" character. For more, see the @lavamoat/aa package`

Reply:
done